### PR TITLE
Add current step prompt for screenreader

### PR DESCRIPTION
**Pull request by avaleske**
# Pull Request Template

## Issue #267

Addresses #267

- [ ] This change addresses the issue in full
- [x] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description
Add a sr only text that says "current step" as you're navigating through the progress menu.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Prototype/POC (not to merge)
- [ ] This change is a refactor/address technical debt
- [ ] This change requires a documentation update
- [ ] This change requires a SQL Script

## How Can This Be Tested/Reviewed?
Tab through the progress bar with a screen reader.

- [ ] Desktop View
- [ ] Mobile View
- [ ] Test A
- [ ] Test B

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [ ] I have updated the changelog to include a description of my changes
- [ ] I have run `yarn generate:client` if I made backend changes

### CODE REVIEW

File: ui-components/src/locales/bn.json [hunk 1]
```diff
@@ -1123,7 +1123,8 @@
         "name": "স্থানীয় অঞ্চল"
     },
     "progressNav": {
-        "srHeading": "অগ্রগতি"
+        "srHeading": "অগ্রগতি",
+        "current": "الخطوة الحالية: "
```
Comment (willrlin) [inline ui-components/src/locales/bn.json:1127]: this looks like arabic, not bengali

File: ui-components/src/locales/ar.json [hunk 2]
```diff
@@ -1123,7 +1123,8 @@
         "name": "المنطقة المحلية"
     },
     "progressNav": {
-        "srHeading": "تقدم"
+        "srHeading": "تقدم",
+        "current": "Current step: "
```
Comment (willrlin) [inline ui-components/src/locales/ar.json:1127]: Probably just filled the wrong one in here?

**Review by willrlin — APPROVED** [review]

File: ui-components/src/locales/ar.json (hunk 2 above)
Comment (avaleske) [reply ui-components/src/locales/ar.json]: yep

File: ui-components/src/locales/bn.json (hunk 1 above)
Comment (avaleske) [reply ui-components/src/locales/bn.json]: fixed

**Pull request merged**